CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 3.9.0 
+
```

Comment:
We should call out the new minimum Terraform version.

```suggestion

NOTES:

* The minimum required Terraform version for this release is Terraform 0.12+

```

Reply:
Which may also be a breaking change ...?

Reply:
In which case this should be v4.0.0. In the past, when we required a more recent Terraform version, did we consider it a breaking change?

Reply:
Strictly, yes. But I foresee a bigger release with breaking changes in the near future. We could cleanup quite a few things here. If by any change possible I'd try to leave it at v3. But yeah, strictly it should be 🤷

Reply:
is this really a new requirement?
this is already in the readme: terraform >= 0.12.x

Reply:
Good catch. It has been changed here: https://github.com/gitlabhq/terraform-provider-gitlab/commit/a1e833223f1067ab3394194369d31e1522432c9a already. We are probably good.